setup.py

```diff
@@ -127,12 +127,21 @@ def get_ifconfig():
 NAME = "wifiphisher"
 AUTHOR = "sophron"
 AUTHOR_EMAIL = "sophron@latthi.com"
+URL = "https://github.com/wifiphisher/wifiphisher"
 DESCRIPTION = "Automated phishing attacks against Wi-Fi networks"
 LICENSE = "GPL"
 KEYWORDS = ["wifiphisher", "evil", "twin", "phishing"]
-PACKAGES = find_packages()
+PACKAGES = find_packages(exclude=["docs", "tests"])
 INCLUDE_PACKAGE_DATA = True
 VERSION = "1.2"
+CLASSIFIERS = ["Development Status :: 5 - Production/Stable",
```

Comment:
[Here](https://pypi.python.org/pypi?%3Aaction=list_classifiers) is the full list of classifiers and these are the possible choices for development status:
>Development Status :: 1 - Planning
Development Status :: 2 - Pre-Alpha
Development Status :: 3 - Alpha
Development Status :: 4 - Beta
Development Status :: 5 - Production/Stable
Development Status :: 6 - Mature
Development Status :: 7 - Inactive